test/TestPercentageMath.sol

```diff
@@ -113,10 +110,7 @@ contract TestPercentageMath is Test {
         vm.assume(p <= PERCENTAGE_FACTOR);
         vm.assume(x <= MAX_UINT256 / (PERCENTAGE_FACTOR - p));
 
-        assertEq(
-            PercentageMath.percentSub(x, p),
-            (x * (PERCENTAGE_FACTOR - p) + HALF_PERCENTAGE_FACTOR) / PERCENTAGE_FACTOR
-        );
+        assertEq(PercentageMath.percentSub(x, p), mathRef.percentSub(x, p));
```

Comment:
small warning here but at the left we're calling directly the lib while at the right we're walling the contract.

Reply:
it is to avoid re-writing the ref two times. what do you prefer ?

Reply:
as you want, but seems weird

Reply:
Ok, I will keep it like this to avoid code duplication